plugins/SLURM-Integration/install.sh

```diff
@@ -64,6 +64,19 @@ function update_slurm_conf()
     fi
 }
 
+function update_python_path_for_ufm_prolog_epilog()
+{
+    python_path=$(which python3)
+    sudo sed -i -e "s#python3_path#$python_path#g" $SLURM_DIR/$UFM_PROLOG_FILE
```

Comment:
lines 70-73 export to function: set_python3_path(path)
then call this function from here:
set_python3_path($SLURM_DIR/$UFM_PROLOG_FILE)
set_python3_path($SLURM_DIR/$UFM_EPILOG_FILE)